pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -138,7 +140,10 @@ def write_pep0(self, peps: list[PEP]):
             ("Abandoned, Withdrawn, and Rejected PEPs", dead),
         ]
         for (category, peps_in_category) in pep_categories:
-            self.emit_pep_category(category, peps_in_category)
+            # For subindices, only emit categories with entries.
```

Comment:
Very nitpicky (sorry!): you've mostly used "sub-index/sub-indices" in prose and `subindex`/`subindices` as variables. Maybe just make them all hyphenless? If not:

```suggestion
            # For sub-indices, only emit categories with entries.
```